src/SLIP10Node.ts

```diff
@@ -352,12 +365,14 @@ export class SLIP10Node implements SLIP10NodeInterface {
    * @param options.derivationPath - The rooted HD tree path that will be used
    * to derive the key of this node.
    * @param options.curve - The curve used by the node.
+   * @param options.network - The network for the node. This is only used for
+   * extended keys, and defaults to `mainnet`.
```

Comment:
This is breaking too if we were assuming testnet before?

Reply:
Made a typo in the PR description. Ethereum uses `mainnet`.